packages/functionals/botpress-hitl/README.md

```diff
@@ -70,12 +70,12 @@ A basic implementation example that shows how easy it is to implement a help req
     })
 
     bp.hear(/HITL_STOP/, (event, next) => {
-      bp.messenger.sendText(event.user.id, 'Human in the loop disabled. Bot resumed.')
+      messenger.sendTextMessage(event.user.id, 'Human in the loop disabled. Bot resumed.')
```

Comment:
@dmytropanontko , @emirotin , `messenger` instance isn't available here. So we should either add  `sendTextMessage` to `bp.messenger` or perform somthing else.

Reply:
Ah. Can expose it as `bp.messenger.api` instead of `bp.messenger._internal`?

Reply:
Yes, I think that could work.